### PR TITLE
ShA/fix bug votes collection null date display

### PR DESCRIPTION
**Pull request by AlekseiiShevchuk**

### CODE REVIEW

File: app/Models/Vote.php [hunk 1]
```diff
@@ -143,4 +143,12 @@ public function subscription($userId)
     {
         return $this->morphMany(Subscription::class, Subscription::$name)->where('user_id', $userId)->first();
     }
+
+    public function getFinishedAtAttribute($value)
+    {
+        if($value == '0000-00-00 00:00:00'){
+            return null;
```
Comment (binaryvo) [inline app/Models/Vote.php]: Better to return empty string

**Comment by binaryvo** [comment]
Please fix remark, update your branch from next and re-open PR

**Comment by AlekseiiShevchuk** [comment]
Done

File: resources/assets/templates/vote/voteItem.hbs [hunk 2]
```diff
@@ -10,9 +10,15 @@
...
-            <span>{{ meta.comments}} Comments</span> <span>{{ meta.likes }} likes</span> <span>{{ meta.numberOfUniqueViews }} unique views</span>
-            <span>Finished at {{model.finished_at}}</span><span>{{model.views}} Views</span>
-
+            <span>{{ meta.comments}} Comments</span>
+            <span>{{ meta.likes }} likes</span>
+            <span>{{ meta.numberOfUniqueViews }} unique views</span>
+            {{#if model.finished_at}}
+            <span>Finished at {{model.finished_at}}</span>
+            {{else}}
+            <span>End date is not defined</span>
```
Comment (binaryvo) [inline resources/assets/templates/vote/voteItem.hbs]: Please replace 'End date is not defined' to 'currently in process', add style class with green text

**Comment by binaryvo** [comment]
bug: [link to pic](http://storage3.static.itmages.com/i/16/0912/h_1473667101_7169155_f357167c7d.png)

**Comment by AlekseiiShevchuk** [comment]
Done

**Comment by binaryvo** [comment]
Still shows 'Finished at ' if no date

**Comment by AlekseiiShevchuk** [comment]
it works correctly on my PC
http://clip2net.com/s/3C935ze
i`ve done repush

**Comment by binaryvo** [comment]
Now it works for me

**Pull request merged**